quickstart/llmd-installer.sh

```diff
@@ -406,6 +409,10 @@ install() {
   log_success "ModelService CRD applied"
 
   create_pvc_and_download_model_if_needed
+  if [[ "${DOWNLOAD_ONLY}" == "true" ]]; then
+    log_info "Option \"-b/--download-only)\" specified, will end execution"
```

Comment:
Drop the right parentheses pls to change from `Option "-b/--download-only)" specified, will end execution` to -> `Option "-b/--download-only" specified, will end execution`